neo/core/event.py

```diff
@@ -256,3 +258,48 @@ def as_quantity(self):
         Return the event times as a quantities array.
         """
         return self.view(pq.Quantity)
+
+    def to_epoch(self, pairwise=False, durations=None):
+        """
+        Transform Event to Epoch.
+
+        This method has three modes of action.
+
+        1. By default, an array of `n` event times will be transformed into
+           an array of `n-1` epochs, where the end of one epoch is the
```

Comment:
```suggestion wording was confusing, partly since epochs consist of times and durations
           `n-1` epochs, where the end of one epoch is the
```